holoviews/plotting/bokeh/callbacks.py

```diff
@@ -152,39 +164,84 @@ def __init__(self, plot, streams, source, **params):
         self.streams = streams
         self.comm = self._comm_type(plot, on_msg=self.on_msg)
         self.source = source
+        self.handle_ids = defaultdict(list)
 
 
     def initialize(self):
         plots = [self.plot]
         if self.plot.subplots:
             plots += list(self.plot.subplots.values())
 
```

Comment:
I posted where it's being changed above, in ``set_customjs`` a callback will merge its own ``handle_ids`` with another callback's ``handle_ids`` if that callback has already been attached to the plot handle (since you can't attach multiple callbacks on one handle).

```
for k, v in self.handle_ids.items():
    cb.handle_ids[k] += v
```